config-templates/config.php

```diff
@@ -823,7 +823,14 @@
      *   'proxy' => 'tcp://proxy.example.com:5100'
      */
     'proxy' => null,
-
+    
+    /*
+     * Username/password authentication to proxy (Proxy-Authorization: Basic)
+     * Example:
+     *   'proxy.auth' = 'myuser:password'
+     */
+    'proxy.auth' => null
```

Comment:
I think we should document the proxy feature in general. Probably the best place to do that is a new section in the [advanced features page](https://simplesamlphp.org/docs/stable/simplesamlphp-advancedfeatures).

We tend to add options to the configuration templates without properly documenting them (and I'm the first to claim responsibility of that :hand: ), and that's causing our users to miss a lot of important information, I think.

Needless to say this won't be a show stopper.